### PR TITLE
Feeture/sqone 178/wp media styles

### PR DESCRIPTION
**Pull request by dpellenwood**
Handles misc media blocks & oEmbeds w/ custom overlay.

[SQONE-178]

[SQONE-178]: https://moderntribe.atlassian.net/browse/SQONE-178

### CODE REVIEW

File: wp-content/plugins/core/src/Theme/Media/Oembed_Filter.php [hunk 1]
```diff
@@ -79,11 +77,12 @@ public function get_video_component( $html, $data, $url ) {
 		return $html;
 	}
 
-	private function get_layout_container_attrs( $provider_name, $embed_id ): array {
+	private function get_layout_container_attrs( $provider_name, $embed_id, $title ): array {
 		return [
 			'data-js'             => 'c-video',
 			'data-embed-id'       => $embed_id,
 			'data-embed-provider' => $provider_name,
+			'data-embed-title'    => $title,
```
Comment (ryanurban) [inline wp-content/plugins/core/src/Theme/Media/Oembed_Filter.php:85]: 🔥 🥇

File: wp-content/themes/core/components/video/js/video.js [hunk 2]
```diff
@@ -99,6 +98,7 @@ const playEmbed = ( e ) => {
 	iframe.height = 720;
 	iframe.tabIndex = 0;
 	iframe.allow = 'autoplay; fullscreen';
+	iframe.title = parent.getAttribute( 'data-embed-title' );
```
Comment (ryanurban) [inline wp-content/themes/core/components/video/js/video.js:101]: ❤️

File: wp-content/themes/core/components/video/Video.php [hunk 3]
```diff
@@ -8,85 +8,73 @@
...
+	public const VIDEO_URL        = 'video_url';
+	public const VIDEO_TITLE      = 'video_title';
+	public const TRIGGER_LABEL    = 'trigger_label';
+	public const TRIGGER_POSITION = 'trigger_position';
+	public const THUMBNAIL_URL    = 'thumbnail_url';
+	public const SHIM_URL         = 'shim_url';
 
-	public const CAPTION_POSITION_CENTER = 'center';
-	public const CAPTION_POSITION_BOTTOM = 'bottom';
-	public const CAPTION_POSITION_BELOW  = 'below';
```
[the remark this answers is not in the document]
Comment (dpellenwood) [reply wp-content/themes/core/components/video/Video.php:36]: The caption (along with the wrapping  `<figure>`) is provided by WP core. It is not necessary within the component any longer. If a dev were to use this outside an oEmbed context, they could compose a wrapper component as a figure and inject this and a figcation within it.

[the remark this answers is not in the document]
Comment (ryanurban) [reply wp-content/themes/core/components/video/Video.php:36]: Right, ah, so you're saying we've already handled the center/bottom of embed wrapper positioning/handling of the WP core caption stuff, and there basically isn't a need to do anything other than make sure the default caption looks like it should if center/bottom aren't selected?

[the remark this answers is not in the document]
Comment (ryanurban) [reply wp-content/themes/core/components/video/Video.php:36]: Ah got it, thanks for the explanation

**Pull request merged**